src/Meditation.AttachProcessService/Models/ProcessId.cs

```diff
@@ -0,0 +1,4 @@
+﻿namespace Meditation.AttachProcessService.Models
+{
+    public record struct ProcessId(int Value);
```

Comment:
[ToFix] Validate arguemnt: ```Value > 0```
(Yeah this will ruin the magic of ```record``` :'))

Reply:
`Process.GetProcesses()` seems to also return the system idle process with PID=0 on Windows.
Added check that it can not be negative.